Open Judge System/Web/OJS.Web/Areas/Administration/Views/Contests/EditorTemplates/CategoryDropDown.cshtml

```diff
@@ -13,21 +14,11 @@
       })
       .HtmlAttributes(new { style = "width:100%" }))
 
-<script>
-    function categoryDropdownTemplate(data) {
-        if (data.Parent == undefined) {
-            return data.Name;
-        }
-        else {
-            return data.Parent + " - " + data.Name;
-        }
-    };
-</script>
```

Comment:
Why has this been removed? Are you sure it doesn't break existing functionality?

Reply:
Yes, it was not used anywhere